### ISSUE TITLE
Conda environment import problem

### ISSUE THREAD

**Issue by krakenkrak** [adbailey1/daic_woz_process#1]
Hi, thank you very much for your work! Could you upload your environment from miniconda using both options below? Now it's impossible to load  your .yml files on my Windows machine. Also in second daic_woz repo, please.

"--no-builds" option to "conda env export"

or something like this:

https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#exporting-an-environment-file-across-platforms

**Comment by adbailey1** [comment]
Hi Krakenkrak,

Thank you for your email. I have been meaning to re-upload my code as I've
done a bit of work to neaten everything over the last few months. My
project moved on which stopped me from making it as presentable
as possible.

With regard to the conda environments, I use the same one for both
frameworks.

I've included an attachment of an updated copy, let me know if it works for
you and if it does, I'll upload this to the repo. Obviously let me know if
it doesn't work and we'll have to figure something else out.

With best regards,

Andrew

**Comment by krakenkrak** [comment]
Hi adbailey1,

Thanks for quick respond, but I can't see any attachment here or in my e-mail. Maybe wetransfer, or something? Here is my email: kacper.elsner@gmail.com

Today I istalled WSL and anaconda in it and now I'm trying to compile it with WSL interpreter in PyCharm, but it's rather hard to me because I'm not python genius or something :)

I have also some more questions:
1. Is unzipping automated here?
2. I don't understand these two first statements in comments in config file:
# Set to complete to use all the data
# Set to sub to use training/dev sets only
To which 'parameter' are these options related and where to modify them?
3. Is this code already able to output reasonable F1 score? What about your results 3 months ago and nowadays? I'm interested only in audio-source calculations, not text stuff.

**Comment by adbailey1** [comment]
Hi Krakenkrak,

I didn't realise this was a comment, I'll drop you an email with the env
file.

Don't worry, I'm not a genius at python, I started about 18 months ago! I
haven't used WSL so I can't comment, however, regarding your other
questions, let me get back to you. I think the updates I have done in the
last few months will make the framework easier to use. For now I can say:

1) In the updated version the train/dev/test csv files given with the
dataset are extracted to make a single csv file and from here you can
create folds to use cross validation.
2) This bit probably needs to be changed, I always set this to sub as I
wanted to use a validation set during training my neural network. This can
be altered in config_1.py under 'MODE': 'sub'
3) The code already outputs the F1 score for both the depressed and non
depressed classes and also computes the mean, although I started work on
adding variations of the F1-score such as macro and micro etc.
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_recall_fscore_support.html

As I said, let me re-work the code and re-upload a better version that
should be easier to use. But any questions in the meantime feel free to ask.

With best regards,

Andrew
